rpc/handlers.go

```diff
@@ -43,6 +47,143 @@ func (s *rpcServer) PostV1ClientBroadcastTxSync(ctx echo.Context) error {
 	return nil
 }
 
+// DISCUSSION: This may need to be changed when the GetSession function is actually implemented
+func (s *rpcServer) PostV1ClientDispatch(ctx echo.Context) error {
+	var body DispatchRequest
+	if err := ctx.Bind(&body); err != nil {
+		return ctx.String(http.StatusBadRequest, "bad request")
+	}
+
+	session, err := s.GetBus().GetUtilityModule().GetSession(body.AppAddress, body.SessionHeight, body.Chain, body.Geozone)
+	if err != nil {
+		return ctx.String(http.StatusInternalServerError, err.Error())
+	}
+
+	application := session.GetApplication()
+	rpcApp := protocolActorToRPCProtocolActor(application)
+
+	rpcServicers := make([]ProtocolActor, 0)
+	for _, serv := range session.GetServicers() {
```

Comment:
Let's use `servicer` instead of `serv`. I think` serv` is way to close to `srv` in CS land.

Ditto elsewhere